jobs/release.py

```diff
@@ -0,0 +1,20 @@
+#!/bin/env python3
+
+import os
+import sys
+
+from avocado.core.job import Job
+
+
+ROOT_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+RELEASE_TESTS_DIR = os.path.join(os.path.join(ROOT_DIR, 'selftests', 'release'))
```

Comment:
Is there a reason for the double `os.path.join` call? I could not see the difference using the first one or not.

Reply:
Nothing besides a mistake! Thanks for catching that.